server/cromwell.py

```diff
@@ -199,12 +200,18 @@ async def cromwell(request):
                 )
 
             cromwell_post_url = CROMWELL_URL + '/api/workflows/v1'
+
+            google_labels = {'compute-category': 'cromwell'}
+            if labels:
+                google_labels.update(labels)
```

Comment:
It's probably fine, but this does allow the user to override `compute-category` to be whatever they want. You could reorder this if that's unintended behaviour.

Reply:
Great point, thanks! I've swapped the order.